MainWindow/MainWindow.cpp

```diff
@@ -2,6 +2,7 @@
 #include <MainWindow/MainWindow.h>
 #include <MdiChild/MdiChild.h>
 #include <QToolBar>
+#include <NewFileDialog/NewFileDialog.h>
```

Comment:
Please use " " instead of < > 
Separate system and local includes by addind one new line.